app/controllers/receive_package.js

```diff
@@ -121,85 +146,124 @@ export default Ember.Controller.extend({
 
   invalidInventoryNo: Ember.computed({
```

Comment:
can we rename this to `isInvalidInventoryNo` ? The name currently makes me thing that the returned value is a number.
(also we can remove the setter :) )

Reply:
Boolean returning functions should be named starting with `is`, i.e `invalidInventoryNo `

Reply:
should be  something like
```javascript
Ember.computed('package.inventoryNumber', function () {
       return this.verifyInventoryNumber(
        this.get("package.inventoryNumber")
      );
})
```
otherwise it won't reactively update to package changes and will require to be triggered manually

Reply:
applies to all the others in this file